### PR TITLE
Migrate send dialog to React - Closes #355

### LINKED ISSUE #355: Migrate send dialog to React

### PR DESCRIPTION
**Pull request by slaweet**
This PR covers the basic Send component.

I created follow-up issues to implement:
- pending transactions #514 
- custom alert dialogs #515 
- click to send functionality in Transactions list and Account component #516 

Closes #355

### CODE REVIEW

File: src/components/send/index.test.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import chai, { expect } from 'chai';
+import sinonChai from 'sinon-chai';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+import SendContainer from './';
+import store from '../../store';
+
+chai.use(sinonChai);
+
+
```
Comment (reyraa) [inline src/components/send/index.test.js]: Pleas remove this line

File: src/utils/lsk.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import BigNumber from 'bignumber.js';
+
+BigNumber.config({ ERRORS: false });
+
+export const fromRawLsk = value => (
```
Comment (reyraa) [inline src/utils/lsk.js]: Now that you've implemented these utils, would be neat if you can use them in `liskAmount` component. Just a suggestion.

Comment (slaweet) [inline src/utils/lsk.js]: Good point. updated.

**Comment by reyraa** [comment]
Used the dropdown to set the maximum amount and this happened. may balance was zero.
screen shot 2017-07-27 at 09 29 57

File: src/components/send/send.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  handleChange(name, value) {
+    this.setState({
+      [name]: {
+        value,
+        error: this.validateInput(name, value),
+      },
+    });
+  }
+
+  validateInput(name, value) {
```
Comment (reyraa) [inline src/components/send/send.js]: Please also check if the amount is not zero

Comment (slaweet) [inline src/components/send/send.js]: Fixed

**Pull request merged**